hudi-client/hudi-spark-client/src/main/scala/org/apache/spark/sql/hudi/SparkAdapter.scala

```diff
@@ -165,7 +165,11 @@ trait SparkAdapter extends Serializable {
   /**
    * Create instance of [[ParquetFileFormat]]
    */
-  def createHoodieParquetFileFormat(appendPartitionValues: Boolean): Option[ParquetFileFormat]
+  def createLegacyHoodieParquetFileFormat(appendPartitionValues: Boolean): Option[ParquetFileFormat]
+
+  def getFilePath(file: PartitionedFile): Path
```

Comment:
There is an existing API with the same functionality: `HoodieSparkPartitionedFileUtils.getPathFromPartitionedFile`